src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -427,9 +428,28 @@ VtolAttitudeControl::is_fixed_wing_requested()
 		to_fw = _transition_command == vehicle_status_s::VEHICLE_VTOL_STATE_FW;
 	}
 
+	if(_abort_transition) {
+		to_fw = false;
+	}
+
 	return to_fw;
 }
 
+/*
+ * Abort transition
+ */
+void
+VtolAttitudeControl::abort_transition()
+{
+	if(!_abort_transition) {
+		// TODO: add mavlink log message
+		_abort_transition = true;
```

Comment:
- This is never reset?
- Should be named _abort_forward_transition (or similar, also the method)